### PR TITLE
Documentation/fw_dell_optiplex.md: update coreboot revision and coreboot-sdk revision

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

**Comment by pietrushnic** [comment]
It is great you were able to achieve some improvements for Dell OptiPlex. What I really miss is the impact on our products. This improvement will be lost in this repo. Why don't we push that to the official Dasharo Dell OptiPlex documentation and not link there, but instead use TrechnBoot documentation?

@BeataZdunczyk @macpijan cc

**Comment by krystian-hebel** [comment]
> Why don't we push that to the official Dasharo Dell OptiPlex documentation

Because this is not Dasharo, this is upstream coreboot.

File: Documentation/fw_dell_optiplex.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ correctly.
     docker run --rm -it \
         -v $PWD:/home/coreboot/coreboot \
         -w /home/coreboot/coreboot \
-        coreboot/coreboot-sdk:0ad5fbd48d
+        coreboot/coreboot-sdk:2023-06-04_44f676afc9
     ```
 
 - Build coreboot firmware:
```
Comment (krystian-hebel) [inline Documentation/fw_dell_optiplex.md:38]: This builds without errors, but I don't think `Built dell/e6400 (Latitude E6400)` is the expected result...

**Review by krystian-hebel — CHANGES_REQUESTED** [review]
Please fix the typo in the commit message. Some summary as to why this bump was needed would also be nice.

File: Documentation/fw_dell_optiplex.md (hunk 1 above)
Comment (miczyg1) [reply Documentation/fw_dell_optiplex.md:38]: Okay... New target landed upstream...

Comment (miczyg1) [reply Documentation/fw_dell_optiplex.md:38]: https://github.com/TrenchBoot/trenchboot-sdk/pull/9/commits/b51841bf25f315aaee1d36657353137aa5cac361

**Comment by pietrushnic** [comment]
> Because this is not Dasharo, this is upstream coreboot.

@krystian-hebel upstream coreboot is the basis for almost all Dasharo code. We should treat the TB AEM project as an opportunity to bring products to the 3mdeb shop, but I understand your point - this is not your job, not part of the scope. So I will assume this should be part of the backlog tasks for Dasharo, compatible with Optiplex release @macpijan @BeataZdunczyk cc

@krystian-hebel, one more thing to be clear in future discussion. Ideally, Dasharo releases would use everything upstream without changes, and the only value the Dasharo Team would add is distribution, compliance (aka certification), and support as the base.

**Comment by SergiiDmytruk** [comment]
[Cleanup](https://github.com/TrenchBoot/trenchboot-issues/issues/48): @krystian-hebel, maybe you want to finish the review.

**Comment by krystian-hebel** [comment]
It was moved to https://docs.dasharo.com/variants/dell_optiplex/building-manual/, and old instructions for building and config file will be removed from here as part of https://github.com/TrenchBoot/trenchboot-issues/issues/29.

**Pull request closed without merging**